### DISCUSSION TITLE
Welcome to Smurfs!

### DISCUSSION THREAD

**Discussion by peacecwz** [Trendyol/smurfs#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alielbashir** [comment]
Hello :wave: I'm Ali, a 4th year CSE student interested in distributed systems. I architected and built a gRPC based system this past summer with Python, Go and Dart services, utilizing both unary and streaming RPCs. 

I learned about this project during @peacecwz 's MicroCLI talk at Developers Day 201, where I asked some questions about the underlying system.

I'd like to contribute to the Go and Python host and client implementations. I'm also interested in implementing the GitHub provider

**Reply by peacecwz** [reply]
Hello, @alielbashir Thank you very much for your interest 🙏 We welcome your contribution. I saw your question in Discussions and I answered it. We would like to continue on it. Thanks again 🙏
